features/steps/user_initialisation_steps.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=E0102, E0602
+from behave import *
+from selenium.webdriver.support.ui import Select
+
+from features.fixtures import get_chrome_browser
+from features.steps.helpers import sign_in
+from users.factories import AdminUserFactory
+from users.factories import UserFactory
+
+
+@given("an admin exists in database")
+def step_impl(context):
+    context.admin = AdminUserFactory()
+
+
+@when("the user signs up")
+def step_impl(context):
+    # TODO: HAVE TO SOLVE CAPTCHA AND EMAIL VERIFICATION ISSUES
+    context.user = UserFactory()
+    sign_in(context)
+
+
+@when("admin changes user's employee type to '{user_type}'")
+def step_impl(context, user_type):
+    other_browser = get_chrome_browser()
+    sign_in(context=context, login=context.admin.email, browser=other_browser)
+    other_browser.find_element_by_link_text("Employees").click()
+    table = other_browser.find_element_by_class_name("table")
+    button = table.find_element_by_xpath(".//tbody/child::tr[3]/child::td[7]//a")
```

Comment:
isn't this `tr[3]`  and `td[7]` a bit risky hardcoding? Couldn't we have some `id` there (or sth else that would make those test less fragile)?

Reply:
From what I understood, according to @rwrzesien the Xpath should be used with majority of elements and that we should assume for testing purposes, that the precise layout is known. He also said, and I quote:

> księga ulicy mówi jasno - zmieniasz kody, zmieniasz testy

Reply:
I love this quote

Reply:
I understand @dybi doubts but I also have another rule - never ever apply code to tests (meaning if something is difficult to test, still test it the hardway, don't change the code).  Why? Because if you start doing this you will be tended to do more and more shortcuts. 

I also think that it won't change that often. HTML should be written in a way that it can be easily modified with CSS for better/different looking. I think we didn't change the order of those elements since they were implemented for the first time? We only added some columns and each of those adds were rather big features.

Still, I think it could be tested in different way, at least for column. You could determine which column index has the element in header (`<thead><tr><th>`) basing on its text, and then apply same index here.

Reply:
Done, except:
The columns containing links don't have any names, so I search for the row containing user name and then I pick the first column after named columns (headers count).